ee/billing/test/test_billing_manager.py

```diff
@@ -152,6 +152,12 @@ def test_update_org_details_preserves_quota_limits(self, patch_capture):
             "feature_flag_requests": {"usage": 25, "limit": 300, "todays_usage": 5},
             "api_queries_read_bytes": {"usage": 1000, "limit": 1000000, "todays_usage": 500},
             "period": ["2024-01-01T00:00:00Z", "2024-01-31T23:59:59Z"],
+            "surveys": {
+                "usage": 10,
+                "limit": 100,
+                "todays_usage": 5,
+                "quota_limiting_suspended_until": 1611705600,
+            },
```

Comment:
It doesn't impact this test probably but logically `usage` should be greater than `limit` if `quota_limiting_suspended_until` is set

Reply:
Ok I'll fix it for this and other tests in a separate PR